docs/source/using/managing-minishift.adoc

```diff
@@ -322,3 +322,52 @@ Enables provisioning the OpenShift link:https://docs.openshift.org/latest/archit
 
 `extra-clusterup-flags`::
 Enables passing flags that are not directly exposed in the Minishift CLI directly to `oc cluster up`.
+
+
+[[hyperv-static-ip]]
+=== Assign IP address to Hyper-V (experimental)
+
+Since the Internal Virtual Switch for Hyper-V does not provide a DHCP offer option, an IP address needs to be provided in a different way.
+On Hyper-V a functionality is provided to assign an IP address on startup using the Data Exchange Service.
+
+[IMPORTANT]
```

Comment:
Could you replace it with `NOTE` and here as well https://github.com/gbraad/minishift/blob/e2e6dab3af183b7660cb877d59a5342f23503751/docs/source/using/managing-minishift.adoc#experimental-features. It is breaking consistency IMO as we have `NOTE` everywhere.

Reply:
Above in the same file IMPORTANT is used. This is not a note, but a block with requirements.